### DISCUSSION TITLE
How do I disable map until it is clicked ?

### DISCUSSION THREAD

**Discussion by billnbell** [react-native-maps/react-native-maps#4424] [Q&A]
### Summary

Click to activate?

When scrolling, if my finger enters the map, I don't want it to scroll until I click it.

How do I handle this?

### Reproducible sample code

```TSX
Question
```

### Steps to reproduce

Question

### Expected result

Question

### Actual result

Question

### React Native Maps Version

1.1.0

### What platforms are you seeing the problem on?

iOS (Google Maps)

### React Native Version

0.69.4

### What version of Expo are you using?

Not using Expo

### Device(s)

Iphone

### Additional information

None

**Comment by pinpong** [comment]
You can use onPress function to track if the map pressed.

Issue can be closed.
